src/remark/parseRedirects.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+
+// Parse _redirects file into Docusaurus global data
+async function plugin(context, options) {
+  return {
+    name: 'parse-redirects-plugin',
+    async loadContent() {
+      const redirectsPath = context.siteDir.concat("/static/_redirects");
+      const contents = fs.readFileSync(redirectsPath, 'utf8');
+      const redirects = {};
+      contents.split('\n').forEach(line => {
+        const [source, destination] = line.split(/\s+/);
+        if (source && destination) {
+          redirects[source.replace(/\/$/, '')] = destination.replace(/\/$/, '');
+        }
+        //TODO: clean up commented lines
```

Comment:
Maybe something like this?

```suggestion
        // Check if the line is not a comment
        if (!line.trim().startsWith("#")) {
          const [source, destination] = line.split(/\s+/);
          if (source && destination) {
            redirects[source.replace(/\/$/, '')] = destination.replace(/\/$/, '');
          }
        }
```